### PR TITLE
Add firstRunID and correct originalRunID

### PR DESCRIPTION
**Pull request by longquanzheng**

### CODE REVIEW

File: service/history/mutableStateBuilder.go [hunk 1]
```diff
@@ -1306,7 +1306,7 @@ func (e *mutableStateBuilder) DeleteSignalRequested(requestID string) {
 
 func (e *mutableStateBuilder) addWorkflowExecutionStartedEventForContinueAsNew(domainID string,
```
Comment (shreyassrivatsan) [inline service/history/mutableStateBuilder.go]: The caller of this method will have to pass in `firstRunID` instead of `originalRunID`..

Comment (shreyassrivatsan) [reply service/history/mutableStateBuilder.go]: or maybe both?

Comment (longquanzheng) [reply service/history/mutableStateBuilder.go]: originalRunID is in execution.GetRunID. I don't think we should pass duplicated info here. We previously called firstRunID as originalRunID but later Max decided to change it to firstRunID and reserve originalRunID as the runID when WF start event is written.

Comment (shreyassrivatsan) [reply service/history/mutableStateBuilder.go]: Ok, makes sense.. We still need to change the caller to pass in the `firstRunID` right?

Comment (longquanzheng) [reply service/history/mutableStateBuilder.go]: I See. Thanks for pointing it out

Comment (meiliang86) [reply service/history/mutableStateBuilder.go]: Can you add comments to these two fields explaining the meaning? This can be easily forget.

Comment (longquanzheng) [reply service/history/mutableStateBuilder.go]: Sure, added to AddWorkflowExecutionStartedEvent

**Pull request merged**